pom.xml

```diff
@@ -106,7 +106,7 @@
         <version.spring>5.0.15.RELEASE</version.spring>
         <version.jetty-server>9.4.11.v20180605</version.jetty-server>
         <version.json-schema-validator>0.1.19</version.json-schema-validator>
-        <version.byte-buddy>1.10.12</version.byte-buddy>
+        <version.byte-buddy>1.10.18</version.byte-buddy>
```

Comment:
[minor] maybe adding another property for the asm-tree dependency just next to this one + with a small comment so we don't forget to keep those two in sync.